Casks/unity-windows-support-for-editor@2017.3.0.rb

```diff
@@ -0,0 +1,14 @@
+cask 'unity-windows-support-for-editor@2017.3.0' do
+  version '2017.3.0f3,a9f86dcd79df'
+  sha256 'b4b6c254f3bcb1c5f62dac44c187e6e4615c3758a2bd570727598e460d325c98'
+
+  url "https://netstorage.unity3d.com/unity/#{version.after_comma}/MacEditorTargetInstaller/UnitySetup-Windows-Support-for-Editor-#{version.before_comma}.pkg"
+  name 'Unity Windows Build Support'
+  homepage 'https://unity3d.com/unity/'
+
+  depends_on cask: 'unity'
```

Comment:
This will conflict with `unity@2017.3.0`, unfortunately.

Reply:
See how our `unity@2017.1.1` avoids that: https://github.com/improbable-io/homebrew-spatialos/blob/master/Casks/unity-windows-support-for-editor%402017.1.1.rb#L9

Reply:
Make sure to propagate this fix to other files, adjusting for their names.

Reply:
good catch, fixed.

Reply:
and tested

<img width="877" alt="screen shot 2018-02-02 at 15 22 05" src="https://user-images.githubusercontent.com/19690292/35741028-9a687e4a-082e-11e8-95e4-bb86cae2a2e1.png">